tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.UnitTests/Contexts/GiasEstablishmentsQueryFilterTests.cs

```diff
@@ -6,20 +6,41 @@ namespace DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.UnitTests.Contexts
 public class GiasEstablishmentsQueryFilterTests
 {
     [Theory]
-    [InlineData("Academies", true)]
-    [InlineData("Colleges", true)]
-    [InlineData("Free Schools", true)]
-    [InlineData("Local authority maintained schools", true)]
-    [InlineData("Special schools", true)]
-    [InlineData("Independent schools", false)]
-    [InlineData("Online provider", false)]
-    [InlineData("Other types", false)]
-    [InlineData("Universities", false)]
+    [InlineData("Academies", "Open", true)]
+    [InlineData("Colleges", "Open",true)]
+    [InlineData("Free Schools", "Open",true)]
+    [InlineData("Local authority maintained schools", "Open",true)]
+    [InlineData("Special schools", "Open",true)]
+    [InlineData("Independent schools", "Open",false)]
+    [InlineData("Online provider", "Open",false)]
+    [InlineData("Other types", "Open",false)]
+    [InlineData("Universities", "Open",false)]
     public void GiasEstablishmentsQueryFilter_should_filter_for_supported_school_types(string schoolType,
+        string establishmentStatusName,
         bool expectedResult)
     {
         var filterFunction = AcademiesDbContext.GiasEstablishmentsQueryFilter.Compile();
 
-        filterFunction(new GiasEstablishment { EstablishmentTypeGroupName = schoolType }).Should().Be(expectedResult);
+        filterFunction(new GiasEstablishment
+                { EstablishmentTypeGroupName = schoolType, EstablishmentStatusName = establishmentStatusName })
+            .Should()
+            .Be(expectedResult);
+    }
+
+    [Theory]
+    [InlineData("Academies", "Open", true)]
+    [InlineData("Academies","Closed", false)]
+    [InlineData("Academies","Open, but proposed to close", true)]
+    [InlineData("Academies","Proposed to open", true)]
+    public void GiasEstablishmentQueryFilter_should_filter_for_establishment_status(string schoolType,
+        string establishmentStatusName,
+        bool expectedResult)
+    {
+        var filterFunction = AcademiesDbContext.GiasEstablishmentsQueryFilter.Compile();
+
+        filterFunction(new GiasEstablishment
+                { EstablishmentTypeGroupName = schoolType, EstablishmentStatusName = establishmentStatusName })
```

Comment:
```suggestion
                { EstablishmentTypeGroupName = "Academies", EstablishmentStatusName = establishmentStatusName })
```
As all of these are "Academies", it could just be popped in here rather than in the `InlineData` to make the test a little clearer